cmd/vochaintest/vochaintest.go

```diff
@@ -265,16 +283,34 @@ func censusImport(host string, signer *ethereum.SignKeys) {
 	log.Infof("Census created and published\nRoot: %x\nURI: %s", root, uri)
 }
 
-func waitUntilNextBlock(mainClient *client.Client) error {
-	h, err := mainClient.GetCurrentBlock()
-	if err != nil {
-		return fmt.Errorf("cannot get current height: %w", err)
+func (mainClient testClient) ensureTxCostEquals(signer *ethereum.SignKeys, txType models.TxType, cost uint64) error {
+	for i := 0; i < retries; i++ {
+		// if cost equals expected, we're done
+		newTxCost, _ := mainClient.GetTransactionCost(models.TxType_SET_ACCOUNT_INFO)
```

Comment:
Do not ignore errors please. Even if they look dummy. Cause afterwards debugging a code that hide some errors is pretty inconvenient.

Reply:
done. i added this snippet in some cases where an error is not frequent
```
		if err != nil {
			log.Warn(err)
		}
```
and log.Debug if the error is more common. in any case the loop block should continue trying another iteration.